cmd/kubeadm/app/util/version.go

```diff
@@ -138,6 +150,7 @@ func splitVersion(version string) (string, string, error) {
 
 // Internal helper: return content of URL
 func fetchFromURL(url string, timeout time.Duration) (string, error) {
+	glog.V(1).Infof("fetching Kubernetes version from URL: %s", url)
```

Comment:
Do we really want to have that with V(1)? maybe with bigger verbosity level ?

Reply:
does 2 sound good?